scripts/generate_json_schemas.sh

```diff
@@ -13,9 +13,9 @@ set -e
 LINKML_ITEMS=(
   "credentials.yaml,CoreIdentityJWTClass,CoreIdentityJWT.json"
```

Comment:
Is there a good ordering for these, now that the ordering shows up in the index file?

Would alphabetical be better than the slightly-arbitrary order we have now?

Reply:
I think alphabetic is better and I am sure the tech writer and accessibility community will agree :-)